cluster/job_once_scheduler.go

```diff
@@ -124,18 +124,18 @@ func (s *JobOnceScheduler) ListScheduledJobs() ([]JobOnceMetadata, error) {
 }
 
 // ScheduleOnce creates a scheduled job that will run once. When the clock reaches runAt, the
-// callback will be called with key as the argument.
+// callback will be called with key and props as the argument.
```

Comment:
is it possible to move `props` to the last parameter for better (inline) typography when invoking?

Reply:
Good suggestion. Added in [f8b73bc](https://github.com/mattermost/mattermost-plugin-api/pull/124/commits/f8b73bc6795e54f407ce5aab7cd58ef003bb1d62).